fecgov/fec-cms
# 3022 Upgrade patch/latest version of lodash.mergewith

## Summary

- Resolves #3022 

Adding lodash.mergewith so we can specify the lastest patch version, which is also the latest release.

## Impacted areas of the application

Because it's only a patch version change, there should be no impact apart from addressing the security issues. Any packages that use lodash.mergewith automatically use the latest patch.

## Screenshots

None

## Related PRs

None

## How to test
- `Pull the branch`
- `npm i`
- Check that the site functions correctly, especially where lodash.mergewith may be used
- Have Snyk scan the site again
____
